stl/src/charconv.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This must be as small as possible, because its contents are
+// injected into the msvcprt.lib and msvcprtd.lib import libraries.
+// Do not include or define anything else here.
+// In particular, basic_string must not be included here.
+
+#include <charconv>
+
+namespace std {
+
+    extern const uint32_t _Large_power_data[] = {0x540be400, 0x00000002, 0x63100000, 0x6bc75e2d, 0x00000005, 0x40000000,

```

Comment:
I believe this definition can be `extern constexpr` - not that it makes a significant difference (all the initializers are obviously literals), but it was originally `static constexpr` within a function.

Reply:
We may need to add `/Zc:externConstexpr` to the build (https://docs.microsoft.com/en-us/cpp/build/reference/zc-externconstexpr?view=msvc-160). AFAIK it still hasn't been enabled under `/permissive-`.

Reply:
I don't like the idea of having `extern constexpr` in `.cpp`, but not in header, and having to change build options just for this. Can we stay with `extern const`?

Reply:
Good points about the weird mismatch and the need for an extra compiler option. After thinking about this, I'm fine with the PR's change, and I withdraw my suggestion. Thanks!